src/gimelstudio/application.py

```diff
@@ -94,7 +94,7 @@ def __init__(self):
         self.saveprojectfile_menuitem = flatmenu.FlatMenuItem(
             file_menu,
             id=wx.ID_ANY,
-            label="{0}{1}".format(_("Save Project..."), "\tCtrl+S"),
+            label="{0}{1}".format(_("Save Project…"), "\tCtrl+S"),
```

Comment:
Just so I know, how are you creating these ellipses, @JohJakob ? Is it a keyboard shortcut, from the system special characters or something else? 🔠

Reply:
It’s a very convenient keyboard shortcut on Mac (`option` + `.`). On Windows, you can press `Alt` and then type the code 0133 to get an ellipsis.

Reply:
Okay, thanks. I have just always used ... It works too. 🤣

Reply:
Yeah, I guess it only makes a difference when using a screen reader 😀